src/components/Header.tsx

```diff
@@ -1,33 +1,65 @@
-import React from "react";
+import React, { useState } from "react";
 import { useCookies } from "react-cookie";
 
-import { Typography, Layout, Button } from "antd";
+import { Typography, Layout, Button, Modal } from "antd";
 
 import logo from "../assets/images/logo.svg";
 import "../styles/Header.css";
 import { withRouter, RouteComponentProps } from "react-router-dom";
 
 const Header: React.FC<RouteComponentProps> = (props) => {
-  const [ cookies, setCookie ] = useCookies(['accessToken']);
+  const [cookies, setCookie] = useCookies(["accessToken"]);
+  const [showExitModal, setShowExitModal] = useState(false);
 
   const onLogoutClick = () => {
-    setCookie('accessToken', '');
+    setCookie("accessToken", "");
     props.history.push("/");
-  }
+  };
+
+  const onLogoClick = () => {
+    console.log("Click");
```

Comment:
Clack

Reply:
Is this console.log supposed to be removed?